cmd/lakefs/cmd/run.go

```diff
@@ -103,9 +103,11 @@ var runCmd = &cobra.Command{
 		cfg := loadConfig()
 		viper.WatchConfig()
 		viper.OnConfigChange(func(in fsnotify.Event) {
-			lvl := viper.GetString(config.LoggingLevelKey)
-			logger.WithField("toLevel", lvl).Info("Changing log level")
-			logging.SetLevel(lvl)
+			lvl := viper.GetString("logging.level")
```

Comment:
It might be a bit slower, but can you get the config struct here instead of calling explicitly to `viper.GetString("logging.level")`?